go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/mongodb/mongodb-atlas-cli
 
 go 1.21
 
+toolchain go1.21.1
```

Comment:
[q] why did we need this?

Reply:
This is probably a leftover from the rebase. Removed.